core/src/main/scala/cats/ApplicativeEval.scala

```diff
@@ -0,0 +1,25 @@
+package cats
+
+import simulacrum.typeclass
+import cats.Eval.always
+
+/**
+ * A type class that allows lifting any value into the applicative
+ * context, with its evaluation being controlled by [[Eval]] and
+ * supporting optional laziness.
+ */
+@typeclass trait ApplicativeEval[F[_]] extends Applicative[F] {
+  /**
+   * Lifts any value into the `F[_]` applicative context, where the
+   * evaluation is controlled by [[Eval]] and can be optionally lazy.
+   */
+  def eval[A](a: Eval[A]): F[A]
```

Comment:
would `pure(a).map(_.value)` be a legit implementation of this? If not, can you add a comment to help people understand the purpose of the type.

If this is not legit, can we point to exactly what law rules it out?